Doc/library/py_compile.rst

```diff
@@ -42,6 +42,13 @@ byte-code cache files in the directory containing the source code.
    is raised.  This function returns the path to byte-compiled file, i.e.
    whatever *cfile* value was used.
 
+   The doraise and quiet arguments determine how errors are handled while
+   compiling file. If quiet is 0 or 1, and doraise is false, the default
+   behaviour is enabled: an error string is written to sys.stderr, and the
```

Comment:
sys.stderr -> \`\`sys.stderr\`\` to be consistent with rest of the documentation.